website/docs/core-concepts/connecting-to-data-sources/authentication/README.md

```diff
@@ -1,63 +1,115 @@
+
+
+ <VideoEmbed host="youtube" videoId="IptCmvKdbog" title="Add new API" caption="Add new REST API"/> 
+
+You can provide the URL and additional information like the Headers, Params, Body, and Pagination. Once you have added code in the body section, you can directly run the query and connect it with widgets. To learn more about configuring APIs, you can refer to the documentation on [configuring APIs](/core-concepts/connecting-to-data-sources/authentication/connect-to-apis).
+
+#### Mock API
+
+Appsmith provides a mock API so you can fetch the data and perform actions using a RESTful interface.
+
+```js
+https://mock-api.appsmith.com/users?page=1
+```
+
+This is a mock API exposed by Appsmith to help you learn API basics. Data in the mock API have the following structure:
+
+```js
+{
+  "next": "https://mock-api.appsmith.com/users?page=2&pageSize=10",
+  "previous": "https://mock-api.appsmith.com/users?page=0&pageSize=10",
+  "users": [
+    {
+      "id": 2,
+      "name": "Jenelle Kibbys",
+      "status": "APPROVED",
+      "gender": "Female",
+      "avatar": "https://robohash.org/quiaasperiorespariatur.bmp?size=100x100&set=set1",
+      "email": "jkibby1@hp.com",
+      "address": "85 Tennessee Plaza",
+      "createdAt": "2019-10-04T03:22:23.000Z",
+      "updatedAt": "2019-09-11T20:18:38.000Z"
+    },
+    {...
+```
 
+<VideoEmbed host="youtube" videoId="DWLF0pNjjuI" title="Using A Sample API " caption="How to use mock API | Example"/>
 
-You’ll be redirected to the create datasource page as shown below:
+ To use these mock APIs,
+
+* Click on **+** next to Datasources.
+* Select **REST API**
+* To ```fetch users``` choose the GET method and enter the following URL:
+
+```js
+ GET 'https://mock-api.appsmith.com/users?page=1'
+```
+
+
+The [API pane](/core-concepts/connecting-to-data-sources/authentication/connect-to-apis) is a REST interface that allows you to create and modify your existing APIs.
+
+
+
+### Creating an authenticated API
+
+Authenticated API is an API that requires authentication before allowing access to its resources or capability. This is done to ensure that only authorized users are able to interact with the API and to protect sensitive data or capability from unauthorized access:
+
+ <VideoEmbed host="youtube" videoId="Uy7ZDviGbtM" title="Add new API" caption="Add new API"/> 
 
-![Create a new Datasource](</img/OAuth__API_Integration__Create_New_DB.png>)
 
-## Configure
+* Click on the **+** icon next to the Datasources and choose **Authenticated API**.
+* Provide the configuration details required to connect to your API.
+* To configure, you can provide the URL and additional information like the headers, parameters, [authentication type](core-concepts/connecting-to-data-sources/authentication/authentication-type), [self-signed certificate](/core-concepts/connecting-to-data-sources/authentication/self-signed-certificates), and more.
```

Comment:
Hey on the call yesterday we decided to write the configuration details here for authenticated API. let's keep this as before,  add the missing ones and cross-reference the ones that have detailed explanation in sub-pages - https://docs.appsmith.com/core-concepts/connecting-to-data-sources/authentication#configure